src/main/java/io/kestra/plugin/gcp/bigquery/Query.java

```diff
@@ -426,6 +436,11 @@ public static class Output implements io.kestra.core.models.tasks.Output {
             description = "Only populated if 'store' is set to true."
         )
         private URI uri;
+
+        @Schema(
+            title = "The informations where the data are stored"
+        )
+        private HashMap<String,String> destinationTable;
```

Comment:
Please use a true object for that with properties, it will allow to show documentation on the available one